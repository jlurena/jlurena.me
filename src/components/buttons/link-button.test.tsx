import { render } from '@testing-library/react';
import { describe, it, expect } from 'vitest';
import LinkButton from './link-button';

describe('<LinkButton />', () => {
  it('renders correctly (snapshot)', () => {
    const { asFragment } = render(
      <LinkButton
        aria-label="Open in new tab"
        href="https://example.com"
        target="_blank"
      >
        Hello
      </LinkButton>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
