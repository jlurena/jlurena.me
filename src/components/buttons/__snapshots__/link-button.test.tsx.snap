// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<LinkButton /> > renders correctly (snapshot) 1`] = `
<DocumentFragment>
  <div
    class="_container_1af848"
  >
    <a
      aria-label="Open in new tab"
      href="https://example.com"
      target="_blank"
    >
      Hello
    </a>
  </div>
</DocumentFragment>
`;
